rfc.tex

```diff
@@ -40,5 +41,5 @@ \subsection{Change Control Board}
 Inside DM we have our own formal CCB that has evolved over the years from the SAT to something we called the Technical Control Team (TCT) but which has now been reorganized as the DM CCB.
 The DM CCB is chaired by the DM Systems Engineer and has representation from senior members of each DM team as well as the DM Project Manager and DM Subsystem Scientist.
 DM CCB uses the same RFC process as described above, but issues to be discussed by the TCT are immediately submitted into the \emph{Flagged} state.
-This is mainly used to discuss changes to the baseline documents and is also used to give formal DM approval to changes that DM are requesting to be made in project level documentation.
-When deemed necessary some RFCs may require a video meeting to allow more direct discussion.
+This is mainly used to discuss changes to the baseline documents and test reports associated with formal milestones, and is also used to give formal DM approval to changes that DM are requesting to be made in project level documentation.
```

Comment:
Formally, no comma is used here.  You could rephrase to avoid the multiple "and"s.